### DISCUSSION TITLE
Can Podman Desktop share into other WSL instances like Debian

### DISCUSSION THREAD

**Discussion by TomSweeneyRedHat** [podman-desktop/podman-desktop#731] [Ideas]
I saw this question on Twitter and as I don't know the answer, thought I'd start a discussion here and leave a link in Twitter.  Here's the tweet:

[Dwayne Bradley ](https://twitter.com/dwaynebradley70)💉💉💉💉
[@dwaynebradley70](https://twitter.com/dwaynebradley70)
Replying to 
[@unixterminal](https://twitter.com/unixterminal)
 
[@Podman_io](https://twitter.com/Podman_io)
 [and 2 others](https://twitter.com/dwaynebradley70/status/1585815444988039169/people)
It needs to be able to share into other WSL instances like Debian. Basically install [@Podman_io](https://twitter.com/Podman_io)
 Desktop in Windows and have `podman` command available in Debian. I couldn’t make that work when I tried it last week. Both [@Docker](https://twitter.com/Docker)
 Desktop and [@Rancher_Labs](https://twitter.com/Rancher_Labs)
 Desktop can do this.

**Comment by benoitf** [comment]
I've already replied on twitter https://twitter.com/florentbenoit/status/1585978179021422593

the issue needs to be fixed on Podman (Machine) on Windows
Discussion already exists on podman tracker
https://github.com/containers/podman/discussions/16130#discussioncomment-3945343
